From: SeongJae Park <sj@kernel.org>
Date: Fri, 14 Feb 2025 17:22:56 -0800
Subject: [PATCH] ==== filter documentation fixup ====

Docs/mm/damon: misc DAMOS filters documentation cleanup

Cleanup DAMOS filters documentation by fixing a typo, document
hugepage_size filter on design, moving logic details from usage to
design, clarify DAMOS filters handling sequence based on handling layer,
and polishing the filters type list summary to be easier to read.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_haacx_e4 | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_haacx_e4

diff --git a/hkml_cv_bogus/hkml_cv_bogus_haacx_e4 b/hkml_cv_bogus/hkml_cv_bogus_haacx_e4
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

